dis_snek/ext/paginators.py

```diff
@@ -1,7 +1,7 @@
 import asyncio
 import textwrap
 import uuid
-from typing import List, TYPE_CHECKING, Optional, Callable, Coroutine
+from typing import List, TYPE_CHECKING, Optional, Callable, Coroutine, Union
```

Comment:
Since we are in Python 3.10 I think you should use `|` (pipe) for typehinting an `Union`
(`Union[int, str]` -> `int | str`)

Reply:
We use Union pretty much everywhere. And from what I remember not sure it would work here? You could give it a try though.

Reply:
It works both, but the way with the `|` is the Python3.10+-way (as far I know).

(This was just an idea, but if you use it everywhere, it would be better if `Union` is used, so that the code style is not broken).

Reply:
We haven't actively enforced any rules around which uninion-hint should be used, irrc both have been used in the lib. At some point we could straighten out which to use, but i don't believe now is the time for that